README.md

```diff
@@ -160,7 +158,3 @@ The endpoint of the Grafana Agent.
 You do not need to set an `endpoint` value if your Grafana Agent is running locally with the default gRPC endpoint (localhost:4317).
```

Comment:
Believe it should be default HTTP at localhost:4318.

Also, check the yml snippet at line 117, it still has the 4317 port.

Reply:
Configuration section still references the env variable `GRAFANA_OTLP_CLOUD_API_KEY` at line 125.  I believe this is no longer valid.

Reply:
API key is unrelated - why should it not be valid anymore?

Reply:
I wasn't sure since the PR comments include the following.

> env vars not supported anymore

Reply:
So I see this reference some of the environment variables.

Reply:
Right :smile: 

This refers to the env vars that are read by the SDK auto-config, not "our" properties.

Reply:
It should read: "Environment variables starting with "OTEL_" are not supported anymore..."

Reply:
General question about the move away from OTEL_ env vars.  Based on the [sdk config doc](https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_endpoint), OTEL will evaluate the endpoint to `v1/<SIGNAL>`, but I see that we need to specifically set that suffix in MetricsOtlpConfig (`v1/metrics`).  Not sure why we need to set here since it seems to be the same?